src/test/scala/org/ergoplatform/settings/ErgoSettingsSpecification.scala

```diff
@@ -17,21 +17,21 @@ class ErgoSettingsSpecification extends ErgoPropertyTest {
     settings.nodeSettings shouldBe NodeConfigurationSettings(StateType.Utxo, verifyTransactions = true, 1000,
       poPoWBootstrap = false, 10, mining = false, Constants.DefaultComplexityLimit, 1.second, useExternalMiner = false, miningPubKeyHex = None,
       offlineGeneration = false, keepVersions = 200, mempoolCapacity = 100000, blacklistCapacity = 100000,
-      mempoolCleanupDuration = 10.seconds, rebroadcastCount = 3, minimalFeeAmount = 0)
+      mempoolCleanupDuration = 10.seconds, rebroadcastCount = 3, minimalFeeAmount = 0, headerChainTimeDiff = 100)
```

Comment:
we are not testing that parameter can be changed. Maybe clone the test cases and use some other value?

Reply:
In these tests we're reading settings from a file and check that expected values being read from it. Values can not be changed. We can check another branch in a way like: 

(settings.nodeSettings == NodeConfigurationSettings(StateType.Utxo, verifyTransactions = true, 1000,
      poPoWBootstrap = false, 10, mining = false, Constants.DefaultComplexityLimit, 1.second, useExternalMiner = false, miningPubKeyHex = None,
      offlineGeneration = false, keepVersions = 200, mempoolCapacity = 100000, blacklistCapacity = 100000,
      mempoolCleanupDuration = 10.seconds, rebroadcastCount = 3, minimalFeeAmount = 0, headerChainTimeDiff = 10)) shouldBe false

but I'm not sure we're checking something new here